core/controllers/base.py

```diff
@@ -69,6 +69,8 @@ def load_template(filename):
         str. The HTML file content.
     """
     filepath = os.path.join(feconf.FRONTEND_TEMPLATES_DIR, filename)
+    if filename == 'index.html':
+        filepath = os.path.join('dist', 'oppia-angular', filename)
```

Comment:
Very frail logic, subjected to change. (based on the fact that index.html is only produced by angular cli and not webpack due to our configurations)

Reply:
Done!